.github/workflows/pythonpackage.yml

```diff
@@ -53,3 +53,18 @@ jobs:
       run: python3 -m pip install -r doc/requirements.txt -e .
     - name: Build docs
       run: make -C doc html
+
+  mypy:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-python@v5
+      with:
+        python-version: 3.12
+        cache: pip
+        cache-dependency-path: |
+          'pyproject.toml'
+          'requirements-dev.txt'
+    - run: pip install -r requirements-dev.txt -e .
+    - run: mypy --config-file pyproject.toml .
```

Comment:
Maybe just silence the return code?

Reply:
That would work as a last resort, but I'd like to actually see the status already while we are in the process of making it pass.